advisories/hackage/spacecookie/HSEC-2025-0004.md

```diff
@@ -0,0 +1,35 @@
+```toml
+
+[advisory]
+id = "HSEC-2025-0004"
+cwe = []
+keywords = ["gopher"]
+
+aliases = []
+related = []
+
+[[affected]]
+package = "spacecookie"
+cvss = "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N/E:U/RL:O/RC:C"
+
+declarations = { "Network.Gopher.Util.santinizePath" = ">= 0.2.0.0 && < 1.0", "Network.Gopher.Util.santinizeIfNotUrl" = ">= 0.2.0.0 && < 1.0", "Network.Gopher.Util.sanitizePath" = ">= 1.0.0.0 && < 1.0.0.3", "Network.Gopher.Util.santinizeIfNotUrl" = ">= 1.0.0.0 && < 1.0.0.3" }
```

Comment:
```suggestion
declarations = { "Network.Gopher.Util.santinizePath" = "(>= 0.2.0.0 && < 1.0) || (>= 1.0.0.0 && < 1.0.0.3)", "Network.Gopher.Util.santinizeIfNotUrl" = "(>= 0.2.0.0 && < 1.0) || (>= 1.0.0.0 && < 1.0.0.3)" }
```

Reply:
No, that's not correct. The functions were both misspelt in `< 1.0`, hence two different entries.

Reply:
Fixed spelling of second entry for `s*IfNotUrl`.